advocate_europe/assets/scss/components/_blog.scss

```diff
@@ -0,0 +1,191 @@
+//blog_index_page and blog_page
+
+.blog-category {
+    font-size: 12px;
+    font-weight: 900;
+    letter-spacing: rem(.8px);
+    color: $ae-middle-gray;
+}
+
+.blog-createdates {
+    font-size: 14px;
+    color: $ae-middle-gray;
+}
+
+.blog-image {
+    img {
+        width: 100%;
+        height: auto;
+    }
+}
+
+
+// ---------------------- blog_index_page -------------------------------
+
+.blogindex {
+    background: linear-gradient(to bottom, $brand-primary rem(162px), #fff 0%);
+}
+
+.equal {
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.blogindex-title {
+    h1 {
+        padding: rem(20px) 0 rem(30px) 0;
+        text-align: center;
+        color: #fff;
+    }
+}
+
+.blogindex-no-entry {
+    margin-top: rem(50px);
+    text-align: center;
+    min-height: rem(250px);
+}
+
+// blog index page entries
+
+.blogindexentry {
+    .row {
+        margin: 0;
+        padding: 0;
+    }
+
+    .blogindex-featured {
+        margin-bottom: rem(30px);
+    }
+
+    .blogindex-all {
+        margin-bottom: rem(10px);
+    }
+
+    .blogindex-text {
+        padding: rem(15px);
+        background-color: $ae-light-gray;
+        box-shadow: 0 1px 0 0 $ae-light-gray-shadow;
+    }
+}
+
+.blog-index-image {
+    background-size: cover;
+    padding: 0;
+    min-height: rem(300px);
+
+    @media screen and (max-width: $screen-lg-min) {
+        min-height: rem(230px);
+    }
+
+    @media screen and (max-width: $screen-md-min) {
+        min-height: rem(200px);
+    }
+}
+
+.blog-index-feature-image {
+    background-size: cover;
+    padding: 0;
+    min-height: rem(500px);
+
+    @media screen and (max-width: $screen-lg-min) {
+        min-height: rem(300px);
+    }
+
+    @media screen and (max-width: $screen-md-min) {
+        min-height: rem(230px);
+    }
+}
+
+.blogindex-dummy-row {
+    margin: 0;
+    position: relative;
+}
+
+// blogindex category buttons
+
+.blogindex-topics {
+    position: absolute;
+}
+
+.blogindex-topics-xs {
+    margin-top: rem(-20px);
+    margin-bottom: rem(10px);
+}
+
+.blogindex-topics-second {
+    position: absolute;
+    top: rem(50px);
+}
+
+.blogindex-topics-xs-second {
+    margin-top: rem(50px);
+    margin-bottom: rem(10px);
+}
+
+.btn-topic {
```

Comment:
This is a rather a remark, than something that should be changed. This is probably a rather specific button, which is intended only for usage on the blog. So I would probably name it something like blogtopic-btn, to prevent name conflicts with other css classes.